python/serialbox/serialization.py

```diff
@@ -39,32 +40,35 @@
             wrapper = wrapper_try
             break
     except Exception as e:
-        #raise
+        # raise
         pass
 
 if wrapper is None:
     raise ImportError("The serialization library could not be found {dirs}".format(dirs=dirs))
 
+
 def type2dtype(typestr, typesize):
     conv = {
-        'int1' : np.int8,
-        'int2' : np.int16,
-        'int4' : np.int32,
-        'int8' : np.int64,
-        'float4' : np.float32,
-        'float324' : np.float32,
-        'double8' : np.float64,
-        'float648' : np.float64
+        'int1': np.int8,
+        'int2': np.int16,
+        'int4': np.int32,
+        'int8': np.int64,
+        'float4': np.float32,
+        'float324': np.float32,
+        'double8': np.float64,
+        'float648': np.float64
```

Comment:
@andyspiros Yeah, I'll align the `np` as the PEP 8 convention doesn't allow spaces before the `:`